accounts-db/src/epoch_accounts_hash.rs

```diff
@@ -23,6 +23,12 @@ impl AsRef<Hash> for EpochAccountsHash {
     }
 }
 
+impl fmt::Display for EpochAccountsHash {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{}", self.0)
+    }
+}
```

Comment:
It can, but why not implement the `Display` trait. Implementing `Display` will allow the `as_ref()` to go away here too
https://github.com/anza-xyz/agave/blob/9d4867e24b8ac4e7ced5e3122ce844a5b56bcd65/runtime/src/bank.rs#L5232

Reply:
Reverted in https://github.com/anza-xyz/agave/pull/2466/commits/17c6f67e6cbb2c4594c663ded8ee12677b09cc36